tests/service.py

```diff
@@ -75,7 +75,7 @@
         "co2": 200.0,
         "longitude": -57.521369,
         "latitude": -25.194156,
-        "recorded": "2020-10-24T20:47:57.370721+00:00",
+        "recorded": "2020-10-24T20:47:57.370721Z",
```

Comment:
curious, why is this change needed?

Reply:
Output changed somewhere after the upgrade, '+00:00' is equivalent to Z anyway.